### ISSUE TITLE
PlayerJoinEvent

### ISSUE THREAD

**Issue by Bloodlex** [ProjectKorra/ProjectKorra#1268]
Hello,

I've seen this weird exception on my server:
Version: Purpur-1845 (MC: 1.19.2)*
ProjectKorra (Core) 1.10.2

```
6.12 16:46:03 [Server] ERROR Could not pass event PlayerJoinEvent to ProjectKorra v1.10.2
16.12 16:46:03 [Server] INFO java.lang.NullPointerException: Cannot invoke "String.equalsIgnoreCase(String)" because "str" is null
16.12 16:46:03 [Server] INFO at com.projectkorra.projectkorra.BendingPlayer.postLoad(BendingPlayer.java:688) ~[ProjectKorra-1.10.2.jar:?]
16.12 16:46:03 [Server] INFO at com.projectkorra.projectkorra.OfflineBendingPlayer.loadAsync(OfflineBendingPlayer.java:99) ~[ProjectKorra-1.10.2.jar:?]
16.12 16:46:03 [Server] INFO at com.projectkorra.projectkorra.BendingPlayer.getOrLoadOfflineAsync(BendingPlayer.java:302) ~[ProjectKorra-1.10.2.jar:?]
16.12 16:46:03 [Server] INFO at com.projectkorra.projectkorra.PKListener.onPlayerJoin(PKListener.java:1198) ~[ProjectKorra-1.10.2.jar:?]
16.12 16:46:03 [Server] INFO at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor279.execute(Unknown Source) ~[?:?]
16.12 16:46:03 [Server] INFO at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
16.12 16:46:03 [Server] INFO at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
16.12 16:46:03 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:678) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
16.12 16:46:03 [Server] INFO at net.minecraft.server.players.PlayerList.postChunkLoadJoin(PlayerList.java:372) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.server.players.PlayerList.lambda$placeNewPlayer$0(PlayerList.java:309) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
16.12 16:46:03 [Server] INFO at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
16.12 16:46:03 [Server] INFO at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1368) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:185) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
16.12 16:46:03 [Server] INFO at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1345) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1338) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.util.thread.BlockableEventLoop.runAllTasks(BlockableEventLoop.java:114) ~[?:?]
16.12 16:46:03 [Server] INFO at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1472) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1194) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:310) ~[purpur-1.19.2.jar:git-Purpur-1845]
16.12 16:46:03 [Server] INFO at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by StrangeOne101** [comment]
This is a weird error. I don't understand how this could ever happen

What PK addons do you have? Including side plugins

**Comment by GeybWas** [comment]
> This is a weird error. I don't understand how this could ever happen
> 
> What PK addons do you have? Including side plugins

This error occurs after using multiabilities and then rejoining the server

**Comment by StrangeOne101** [comment]
While I can't replicate the error exactly, your deleted comment was helpful. I was able to find that relogging quickly with a multiability clears your binds. It's possible this was the cause.

**Comment by StrangeOne101** [comment]
Please join our discord and see if your issue is resolved in the latest beta build. I believe it should be fixed :)

**Comment by StrangeOne101** [comment]
Fixed in 1.11.0

**Comment by Bloodlex** [comment]
> Fixed in 1.11.0

I'm sorry, I must have omitted this topic somehow, thank you for fixing it. I need to try it out 🙂 Great job with the plugin by the way!
